cirq-core/cirq/ops/common_channels.py

```diff
@@ -19,9 +19,10 @@
 
 import numpy as np
 
-from cirq import protocols, value
+from cirq import protocols, value, _compat
```

Comment:
Is `_compat` needed here?

Reply:
pylint doesn't flag this?

Reply:
it did I just was an idiot and missed it.